plover/oslayer/xkeyboardcontrol.py

```diff
@@ -165,39 +165,38 @@ def __init__(self, name='xev'):
         self._display = display.Display()
         self._pipe = os.pipe()
         self._display_lock = threading.Lock()
+        self._readfds = (self._pipe[0], self._display.fileno())
 
     def _on_event(self, event):
         pass
 
+    @with_display_lock
+    def _process_pending_events(self):
+        for __ in range(self._display.pending_events()):
```

Comment:
The double underscore looks weird...?

Reply:
Yes, usually `_` is used to indicate a throw away variable, but it's also the default global shortcut for the gettext builtin, so...